Documented the SSUI updates

suyogsainkar opened the pull request:
Documented the SSUI updates teamed with new screen captures. Also made structural changes (new sections + moved some of the existing content) to the Introduction to the Self Service UI guide for better user experience.
I've also documented an overview of the Cockpit Integration with relevant screen captures to this PR.

@dayleparker Dayle, could you please review and merge this PR if you're happy with the changes? Do let me know if you have any comments. Updates in this PR are relevant to bugs 1381450 and 1382221.

Thanks,
Suyog

dayleparker commented:
Hi Suyog,  
Thanks for the PR. Since you made quite a lot of changes to the SSUI guide, I read through the whole book, and I think it flows and reads great! :) I think there's a lot of good description and really clear explanations about what parts of the SSUI do. 

I only have a few small comments -- please assess as you wish and send me back this PR once you've had a look and made any changes you choose to (some of the things that stuck out to me may be intentional decisions, so you can disregard if so). I realize some of my comments might not be related to your work for this PR, as I reviewed the entire guide...

**Comments**

*Chapter 3:*

* This sentence reads a little awkwardly to me -- it sounds like an argument: "Besides, you can see the currently logged in LDAP user, for example Administrator." --> perhaps "Besides that, you can see the currently logged in LDAP user, for example Administrator." or "Additionally, you can see the currently logged in LDAP user, for example Administrator." I would also add a comma before "Administrator".

* At first, I couldn't understand what the screenshot showing changing groups was about. I think this could be fixed by being moved to before the "Note" about clicking the arrow icon, so it's right below the text about changing groups.
* The note about clicking the arrow icon seems out of place in its current location. And it's actually duplicated in chapter 4. I think it should be removed from either chapter 3 or chapter 4.

*Chapter 4*
* The note about clicking the arrow icon, the paragraph about changing groups, and the screenshot to change groups are all duplicated here from chapter 3. Was that intentional? I think it would be better to remove the duplication from this short guide, maybe add an xref if desired.

*Section 5.1 Cockpit Integration*
* Please spell out "administrative" in --> "Cockpit is an interactive Linux server admin interface."
* Second sentence: I've remove "and so on" as it's quite informal. Using "such as" implies there could be more tasks anyway. Suggestion for rewording (reword as you wish though!): "You can perform simple administration tasks such as starting containers, storage administration, network configuration, and inspecting logs."
* In step 3 of "5.1.1. Accessing the Cockpit Interface", I suggest a bit of rewording to make the language of the second sentence more direct/simple. Something perhaps like: "Click the Cockpit button located next to the HTML 5 console button to open the Cockpit interface for the virtual machine. A notification at the top of the screen will appear to show the console is loading." (What are your thoughts?)

Thanks so much and nice work!
Cheers,
Dayle

suyogsainkar commented:
@dayleparker Many thanks for your review and suggestions - I've incorporated the edits suggested edits. The duplicate note was unintentional. With the restructuring, I moved it under the navigating the UI chapter (and missed deleting it from its previous place) as that is where I have covered all navigation/UI elements. The rest of the chapters describe the different tabs and their associated workflow. Thanks!
